src/frontend/Komponenter/Behandling/Modal/HenleggModal.tsx

```diff
@@ -34,9 +46,28 @@ export const HenleggModal: FC<{ behandling: Behandling }> = ({ behandling }) =>
         nullstillIkkePersisterteKomponenter,
         settIkkePersistertKomponent,
     } = useApp();
+    const { vergemål, fullmakt } = personopplysninger;
     const [henlagtårsak, settHenlagtårsak] = useState<EHenlagtårsak>();
+    const [huketAvSendBrev, settHuketAvSendBrev] = useState<boolean>(true);
```

Comment:
Det kan være greit å prefixe booleans med "er", "har", eller "skal" for konvensjon. Så ville renamet til `harHuketAvSendBrev`